.travis.yml

```diff
@@ -32,9 +33,19 @@ before_install:
   - "./tests/before_install.sh"
 
 before_script:
-- "./tests/before_script.sh"
+  - "./tests/before_script.sh"
 script:
-  - if [ "${TRAVIS_PHP_VERSION:0:3}" = "$CS" ] && [ "$GLPI_BRANCH" = "$AFTER_SUCCESS_BRANCH" ]; then COVERAGE="--nccfc CommonTreeDropdown CommonDropdown CommonDBTM CommonGLPI"; else COVERAGE="-ncc"; fi
+  - cd plugin/flyvemdm
+  - if [ "${TRAVIS_PHP_VERSION:0:3}" = "$CS" ] && [ "$GLPI_BRANCH" = "$AFTER_SUCCESS_BRANCH" ]; then COVERAGE="--nccfc CommonTreeDropdown CommonDropdown CommonDBTM CommonGLPI"; else COVERAGE="-ncc"; fi 
+  - if [ -e scripts/cliinstall.php ] ; then php ../../scripts/cliinstall.php --db=$OLDDBNAME --user=root --tests ; fi
+  - if [ -e tools/cliinstall.php ] ; then php  ../../tools/cliinstall.php --db=$OLDDBNAME --user=root --tests ; fi
+  - php ../fusioninventory/scripts/cli_install.php --tests
+  - mysql -u root $OLDDBNAME < tests/plugin_flyvemdm_empty_2.0.0-rc.1.sql
+  - mysql -u root $OLDDBNAME < tests/plugin_flyvemdm_config_2.0.0-rc.1.sql
+  - php tools/cli_install.php --test
+  - rm ../../tests/config_db.php
+  - if [ -e scripts/cliinstall.php ] ; then php ../../scripts/cliinstall.php --db=$DBNAME --user=root --tests ; fi
+  - if [ -e tools/cliinstall.php ] ; then php ../../tools/cliinstall.php --db=$DBNAME --user=root --tests ; fi
```

Comment:
install GLPI and Fusion in a DB dedicated to upgrade process
install GLPI in a DB dedicated to fresh install

unit tests will compare the CREATE TABLE statements of all tables of the plugin between the 2 databases, like GLPI does.